src/compose/ComposeBox.js

```diff
@@ -449,7 +449,10 @@ class ComposeBox extends PureComponent<Props, State> {
 
     return (
       <View style={this.styles.wrapper}>
-        {/* $FlowFixMe - `MentionWarnings` should use a type-checked `connect` */}
+        {/*
+          `MentionWarnings` should use a type-checked `connect`
+          $FlowFixMe[incompatible-use]
+        */}
```

Comment:
Here's one remaining comment that gets out of order.

Reply:
Ah, indeed, thanks (and for the other one too).